setup.py

```diff
@@ -92,17 +99,11 @@ def read_utf8(filename):
 )
 
 
-# CI-specific test utilities, e.g. travis, appveyor
-setup_kwargs['extras_require']['ci_test'] = (
-    setup_kwargs['tests_require']
-    + [
-        'codecov',
-        'httpie',
-    ]
-)
```

Comment:
I thought that we should not keep CI-specific dependencies in setup.py, rather move them to `.travis.yml`